include/max_flow/pgr_maxflow.hpp

```diff
@@ -142,12 +142,14 @@ class PgrFlowGraph {
      void set_supersink(
              const std::set<int64_t> &sink_vertices);
 
+     void sort_edges(
```

Comment:
Whatever you are doing as a fix its not working:
https://github.com/pgRouting/GSoC-pgRouting/runs/899643812#step:6:114

Its not building, so:
return the code to what it was and wrap the failing tests with todo.

This is not a quick fix so you are not doing that fix (at least not during the GSoC program)